### DISCUSSION TITLE
Discussing Issue #1636 - Transition class seems irrelevant when creating scale transitions

### DISCUSSION THREAD

**Discussion by artursanntos** [tailwindlabs/tailwindcss#11621] [Help] [answered]
I am opening this discussion as recommended by @adamwathan to continue the discussion about the transition property. The issue is this one: https://github.com/tailwindlabs/tailwindcss.com/issues/1636#issue-1803671602. @thecrypticace clarified that the transitions property is not strictly necessary, but it is a good practice, because you limit the animation effects to properties that can be animated.

@thecrypticace asked for me to share the repository of my code about the duration property not overriding the default duration inside the transition, but i won't be able to share the full code. Instead, i'd like to share the block of code that i got this behavior and if anything else is required to understand the behaviors, i can provide the information.

The code is for the footer in my webpage and it has at least one logo. I wanted to create a scale up effect when hovering, but whenever i put the transition property, the duration property was overriden by the duration inside the transition property. I inspected the elements and it clearly showed that the duration-500 was not being applied and when i deactivated the transition property, it started working. Just to clarify, i am pretty new to tailwind, i have been working with it for only a couple weeks now.
```
{% load static wagtailcore_tags wagtailimages_tags i18n includes_tags %}
{% footer_snippet as footer %}
{% wagtail_site as current_site %}

<footer class="bg-gray text-white border-t-2 border-yellow-500 leading-10">
    <div class="container mx-auto flex flex-col gap-3">
        <div class="grid grid-cols-4 divide-x-2 auto-cols-auto m-3 justify-items-center">
            <div class="py-3 px-10">
                <span class="text-md font-regular font-roboto flex gap-4">
                // Here's the element
                    <a href="https://www.url.com" class="transition ease-in-out duration-500 hover:scale-105">
                        <img src="{% static 'includes/images/logo.png' %}" alt="logo" />
                    </a>
                    {% if footer.logo_footer %}
                        <a href="{{ current_site.root.localized }}">
                            {% image footer.logo_footer original as logo %}
                            <img src="{{ logo.url }}" alt="{{ logo.alt }}" class="transition ease-in-out duration-500 hover:scale-105"/>
                        </a>
                    {% endif %}
                </span>
            </div>
            ... // The rest of the code is not really relevant
```
Does anyone have any idea what is going on?

**Comment by wongjn** [comment, marked as answer]
Seems to work as you'd like in [Tailwind Play](https://play.tailwindcss.com/Ctz5lStrbE). This often indicates the undesired behavior is not caused by Tailwind but something to do with your project, such as a misconfiguration or some other code that may manipulate the CSS after Tailwind.

Consider providing a remotely-hosted git repository that reproduces the undesired behavior. Since you say you cannot share the full code of your project, consider sharing a new project that is similar to your project and that exhibits the undesired behavior.
